index.html

```diff
@@ -1,253 +1,229 @@
+    </div>
 
-          <button class="primary-button">start collaboration</button>
-        </form>
-      </div>
-    </section>
-    <footer class="footer-section">
-      <a class="get-resume-text" href="#"><div>Get my Resume</div> <img src="/images/ic_link_teal.png" alt="" /></a>
-      <div class="footer-social-links">
-        <a href="#"><img src="/images/ic_stackoverflow.png" /></i></a>
-        <a href="#"><img src="/images/ic_github.png" /></i></a>
-        <a href="#"><img src="/images/ic_twitter.png" /></i></a>
-        <a href="#"><img src="/images/linkedin.png" /></i></a>
-        <a href="#"><img src="/images/ic_medium.png"/></i></a>
-        <a href="#"><i class="fa fa-angellist " id="icon-footer"></i></a>
+    <div class="project-container">
+      <img src="./images/project-six.jpg" alt="project-one-image" />
+      <div class="project-bottom-section">
+        <h3>Project name goes here</h3>
+        <ul class="languages-container">
+          <li>HTML/CSS</li>
+          <li id="ruby-text">Ruby on Rails</li>
+          <li>JavaScript</li>
+        </ul>
+        <a href="#">
+          see this project
+          <i class="material-icons" >arrow_forward</i>
+        </a>
       </div>
-    </footer>
-  </body>
-</html>
+    </div>
+    </div>
+  </section>
+  <section class="about-section">
+    <div class="about-info-container">
+      <h2 class="title-text" id="about-text">About</h2>
+      <p class="descriptive-text">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+        minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+        aliquip ea eiusmod
+      </p>
+      <a href="#"  id="get-cv-btn">Get my cv<span class="material-symbols-sharp">
+        download
+        </span></a>
+    </div>
+    <div class="about-languages-container">
+
+      <ul>
+        <p>Languages</p>
+        <li>JavaScript(ES5/ES6)</li>
+        <li>Ruby</li>
+        <li>HTML</li>
+        <li>CSS</li>
+      </ul>
+      <img src="/images/ic_languages_mob.png" alt="" />
+    </div>
+    <div class="framework-container">
+
+      <ul>
+        <p>Frameworks</p>
+        <li>Bootstrap</li>
+        <li>Ruby on rails</li>
+        <li>RSpec</li>
+        <li>Capibara</li>
+        <li>Selenium</li>
+      </ul>
+      <img src="/images/diamond-shape.png" alt="" srcset="" />
+    </div>
+    <div class="skills-container">
+
+      <ul>
+        <p>Skills</p>
+        <li>Codekits</li>
+        <li>Codepen</li>
+        <li>GitHub</li>
+        <li>GitLab</li>
+        <li>Terminal</li>
+      </ul>
+      <img src="/images/ic_skills.png" alt="" />
+    </div>
+  </section>
+  <section class="contact-section">
+    <img src="/images/image_geometry_1.png" alt="" />
+    <div class="contact-form-info-container">
+      <h1 class="title-text">Get started</h1>
```

Comment:
- [ ] As required by the first reviewer you should not have more than one h1 tag on your file, kindly change it to an h2 or h2.